openlibrary/macros/ReadingLogDropper.html

```diff
@@ -143,40 +143,7 @@ <h3>$_('Add to List')</h3>
     </div>
   </div>
   $if include_lists and render_once('lists/widget.addList'):
```

Comment:
Then we can also remove this

```suggestion
  $if include_lists:
```